gulp/tasks/flow.js

```diff
@@ -1,11 +1,30 @@
 var gulp = require('gulp');
 var execFile = require('child_process').execFile;
 
-gulp.task('flow-win', function() {
-  execFile('./flow/flow.exe', ['check','--lib','./flow/libs','--strip-root','./flow'],function (err, stdout, stderr) {
-    if(err) console.log('Error:' + err);
-    if(stdout) console.log(stdout);
-  });
+gulp.task('flow', function() {
+  if (process && process.platform === 'win32') {
```

Comment:
win64 isn't valid. platform will be one of:-

'darwin', 'freebsd', 'linux', 'sunos', 'win32'

https://nodejs.org/api/process.html#process_process_platform